src/quo/components/bottom_sheet/view.cljs

```diff
@@ -73,100 +73,104 @@
-                                                             :to       close-snap-point
-                                                             :clock    clock
-                                                             :easing   (:ease-out animated/easings)
-                                                             :duration close-duration}))
-                          (animated/cond* (animated/not* (animated/clock-running clock))
-                                          [(animated/set manual-close 0)
-                                           (animated/set manual-open 0)
-                                           (animated/call* [] on-close)])])]))
-     [on-cancel])
-    (animated/code!
-     (fn []
-       (animated/cond* (animated/and* (animated/not* manual-close)
-                                      (if @visible 1 0)
-                                      (if (> @height min-height) 1 0))
-                       [(animated/stop-clock clock)
-                        (animated/set open-snap-point (* -1 sheet-height))
-                        (animated/set manual-open 1)]))
-     [@height @visible])
+        ;        on-snap (fn [pos]
+        ;                  (when (= close-snap-point (aget pos 0))
+        ;                    (on-close)))
+        ;        interrupted (animated/and* (animated/eq master-state (:began gesture-handler/states))
+        ;                                   (animated/clock-running clock))
+        ;        translate-y (react/use-memo
+        ;                     (fn []
+        ;                       (animated/with-easing
+        ;                        {:value          (animated/cond* (animated/less-or-eq
+        ;                        master-translation-y 0)
+        ;                                                         (animated/divide master-translation-y
+        ;                                                         2)
+        ;                                                         master-translation-y)
+        ;                         :velocity       master-velocity-y
+        ;                         :offset         offset
+        ;                         :state          master-state
+        ;                         :animation-over drag-over
+        ;                         :snap-points    [open-snap-point close-snap-point]}))
+        ;                     [])
+        opacity 0.3
+        ;        (react/use-memo
+        ;                 (fn []
+        ;                   (animated/cond*
+        ;                    open-snap-point
+        ;                    (animated/interpolate
+        ;                     translate-y
+        ;                     {:inputRange  [(* open-snap-point opacity-coeff) 0]
+        ;                      :outputRange [1 0]
+        ;                      :extrapolate (:clamp animated/extrapolate)})))
+        ;                 [])
+       ]
+    ;    (animated/code!
```

Comment:
Is the intention to merge commented out code like this? As you said, there are many atrocities in the PR, as you mentioned yourself.

Reply:
yeah @ilmotta : all of this was done just to get the builds to work.
I'll clean this up and nuke the commented code out once I fix all issues from QA, there are just a few remaining atm.